buildSrc/src/main/kotlin/maven-publish-conventions.gradle.kts

```diff
@@ -114,22 +79,12 @@ afterEvaluate {
                     }
                 }
             }
-
-            repositories {
-                maven {
-                    url = if (isReleaseBuild()) URI(releaseRepositoryUrl) else URI(snapshotRepositoryUrl)
-
-                    credentials {
-                        username = getRepositoryUsername()
-                        password = getRepositoryPassword()
-                    }
-                }
-            }
         }
+    }
 
-        signing {
-            setRequired({ isReleaseBuild() && gradle.taskGraph.hasTask("publishing") })
-            sign(publishing.publications)
-        }
+    signing {
+        setRequired({ isReleaseBuild() && gradle.taskGraph.hasTask("publishing") })
+        useInMemoryPgpKeys(signingPrivateKey, signingPassword)
+        sign(publishing.publications)
     }
-}
```

Comment:
don't you need?
```
// Fix Gradle warning about signing tasks using publishing task outputs without explicit dependencies
// https://github.com/gradle/gradle/issues/26091
tasks.withType<AbstractPublishToMaven>().configureEach {
    val signingTasks = tasks.withType<Sign>()
    mustRunAfter(signingTasks)
}
```

Reply:
This is not needed because it has not a multiplatform structure.